src/xdocs/config_filters.xml

```diff
@@ -648,6 +648,220 @@ public class UserService {
       </subsection>
     </section>
 
+    <section name="SuppressionXpathFilter">
+      <subsection name="Description">
+        <p>Since Checkstyle 8.4</p>
+        <p>
+          Filter <code>SuppressionXpathFilter</code> works as
+          <code>SuppressionFilter</code>.Additionally, filter
```

Comment:
instead of "code" please make HTML link to filter documentation

Reply:
done